docs/UserGuide.md

```diff
@@ -79,16 +79,49 @@ Adds a client to the address book.
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <box type="tip" seamless>
+    Parameters
+</box>
+
+1. `INDEX` : Must be a valid client index in the filtered list that is currently shown.
+2. `NAME` : Must be alphanumeric, can include spaces.
+3. `PHONE_NUMBER` : Must be numeric and at least 3 digits long.
+4. `EMAIl` : Must ba a valid email address following the format `local-part@domain`.
+5. `ADDRESS`: Can take any values.
+6. `[t/TAG]`: Must be alphanumeric.
+
+
+<box type="warning" seamless>
+    <span circle slot="icon"><md>:bulb:</md></span>
+    Note the following:
+</box>
+
+* If the `INDEX` is invalid, the user will be informed with an error message.
+* Parameters `NAME`, `PHONE_NUMBER`, `EMAIl`and `ADDRESS` cannot be blank.
+* A client can have any number of tags (including 0).
+* Phone number can take on multiple formats, including Singapore and foreign
+  numbers (without the + symbol). This supports insurance agents with an international customer base.
```

Comment:
maybe to add onto this, we can put in the known issues section that the currency of claim may not always be singapore and that the possible remedy is to convert by yourself first before putting into the app?

Reply:
Good point, but did you refer to the wrong part? This part is talking about phone numbers 😅

Reply:
Sures!